web_src/css/base.css

```diff
@@ -521,8 +521,6 @@ a.label,
   color: var(--color-text);
   user-select: auto;
   line-height: var(--line-height-default); /* fomantic uses "1" which causes overflow problems because "1" doesn't consider the descent part */
-  padding-top: 11px; /* counteract line-height change */
-  padding-bottom: 11px; /* counteract line-height change */
```

Comment:
This will cause these elements to be slightly bigger than fomantic default, probably not an issue but I thought to restore previous heights with these, that's why I added them.